cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -110,6 +110,9 @@ type MasterConfiguration struct {
 
 	// The cluster name
 	ClusterName string
+
+	// NotInstallKubeProxy indicates if the kube-proxy will be installed.
+	NotInstallKubeProxy bool
```

Comment:
best to call it `InstallKubeProxy`.

Reply:
@neolit123 As what we discussed in #60428, using `InstallKubeProxy`(just like `readOnly` in #60428) will make us maintain the complicated default value logic...

Reply:
understood, in that case we should probably call it `SkipKubeProxyInstall` or `SkipKubeProxyInstallation`.

Reply:
@neolit123 yep, `SkipKubeProxyInstall` is better for me. 👍

Reply:
:+1: to SkipKubeProxyInstall here.